mlir/utils/jenkins/Dockerfile.rocm

```diff
@@ -101,3 +101,7 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   apt-get clean && \
   rm -rf /var/lib/apt/lists/*
 
+# Copy rocm_agent_enumerator
```

Comment:
I'd prefer to you file a ticket and add the ticket number to this comment that we will remove this once your changes on the specific PR get released.